### DISCUSSION TITLE
About how I can build the current project into a project on a win system to debug with vscode or vs

### DISCUSSION THREAD

**Discussion by nuoyanli** [OpenModelica/OpenModelica#10847] [Q&A]
Hi developer, I'm trying to build and debug in vscode or vs on a win system, but I'm having some problems, is there any information about this?
I've read that there is a script to build OpenModelical into a vs project, but the latest one seems to have removed that part.

**Comment by phannebohm** [comment]
Perhaps this is what you are looking for: https://github.com/OpenModelica/OpenModelica/blob/master/README.cmake.msvc.md
